ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -299,12 +303,35 @@ public enum CocoaPodUtils {
     target 'FrameworkMaker' do\n
     """
 
+    var versionsSpecified = false
+
     // Loop through the subspecs passed in and use the actual Pod name.
     for pod in pods {
-      let version = pod.version == nil ? "" : ", '\(pod.version!)'"
-      podfile += "  pod '\(pod.name)'" + version + "\n"
+      podfile += "  pod '\(pod.name)'"
+      // Check if we want to use a local version of the podspec.
+      if let localURL = LaunchArgs.shared.localPodspecPath,
+        FileManager.default.fileExists(atPath: localURL.appendingPathComponent(pod.name + ".podspec").path) {
+        podfile += ", :path => '" + localURL.path + "'"
+      } else if pod.version != nil {
+        podfile += ", '\(pod.version!)'"
+      }
+      podfile += "\n"
+      if pod.version != nil {
+        versionsSpecified = true
```

Comment:
Can we also assign this up in the block above? Presumably if there's a local podspec there wouldn't also be a version associated with it (and can't be, I think)